reference/6/About/about_Special_Characters.md

```diff
@@ -66,6 +68,25 @@ The output from this command is as follows:
 
 back out
 
+ESCAPE (`e)
+The escape character is most commonly used to specify virtual terminal
+sequences to modify the color of text in addition to other text attributes
```

Comment:
Can't we have both? E.g., "virtual terminal sequences (ANSI escape sequences)"? My only concern is that "virtual terminal sequence" is a lesser-known term that seems to be primarily used in the Windows world - personally, I only recognized it because of the context here.